### ISSUE TITLE
Gitlab NGINX config

### ISSUE THREAD

**Issue by jpoa** [gitlabhq/gitlabhq#8536]
On gitlabhq/lib/support/nginx/gitlab-ssl and gitlabhq/lib/support/nginx/gitlab should it not have

listen [::]:80 ipv6only=on default_server;

on line 42 and 54 (gitlab-ssl) and 38 (gitlab)?

More info here: http://wiki.nginx.org/HttpCoreModule#listen

"The correct way to to this is by using the "ipv6only=on" option in the IPv6 listen directive and also specifying a IPv4 listen directive in the respective server block."

I think its saner to have the two listeners specifically listed (as it is now on the gitlab files), but add the extra parameter to avoid the error.

Basically (as far as I understood) the IPv6 listener will also start an IPv4 listener on Linux, and with the current config we start 1xIPv6 and 2xIPv4 = Error.

I was having a server refusing to start nginx due to this, so I can imagine someone else will stumble on it too.

**Comment by Wachiwi** [comment]
Would you open an PR to patch this issue?

**Comment by jpoa** [comment]
It's here: https://github.com/gitlabhq/gitlabhq/pull/8538

**Comment by jvanbaarsen** [comment]
We are currently in the process of closing down the issue tracker on GitHub, to prevent duplication with the GitLab.com issue tracker. Since this is an older issue I'll be closing this for now. If you think this is still an issue I encourage you to open it on the [GitLab.com issue tracker](https://gitlab.com/gitlab-org/gitlab-ce/issues).
